### PR TITLE
fix(image): Remove duplicate images

### LINKED ISSUE #110: bug(server): Duplicate images in DB when calling `/images` list
## Description

```
(base) ➜  envd git:(166978c) ✗ ./bin/envd image ls                           
NAME                                    CONTEXT                                                                         GPU     CUDA    CUDNN   IMAGE ID        CREATED         SIZE  
gaocegege/test-envd:user-envd-config    /home/gaocegege/code/go/src/github.com/tensorchord/envd/examples/python-basic   false   <none>  <none>  d7d1918bc6b6    4 weeks ago     357MB
gaocegege/test-envd:user-envd-config    /home/gaocegege/code/go/src/github.com/tensorchord/envd/examples/python-basic   false   <none>  <none>  d7d1918bc6b6    4 weeks ago     357MB
gaocegege/test-envd:user-envd-config    /home/gaocegege/code/go/src/github.com/tensorchord/envd/examples/python-basic   false   <none>  <none>  d7d1918bc6b6    4 weeks ago     357MB
gaocegege/test-envd:user-envd-config    /home/gaocegege/code/go/src/github.com/tensorchord/envd/examples/python-basic   false   <none>  <none>  d7d1918bc6b6    4 weeks ago     357MB
```

All these 4 images are the same.

## Additional Info

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

### PR DESCRIPTION
**Pull request by gaocegege**
Fix #110 

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

**Review by VoVAllen — APPROVED** [review]
lgtm

File: pkg/server/environment_create.go [hunk 1]
```diff
@@ -5,15 +5,12 @@
 package server
 
 import (
-	"context"
+	"errors"
```
Comment (kemingy) [inline pkg/server/environment_create.go:8]: use cockroachdb/errors

File: pkg/server/image_get.go [hunk 2]
```diff
@@ -5,17 +5,13 @@
 package server
 
 import (
-	"context"
+	"errors"
```
Comment (kemingy) [inline pkg/server/image_get.go:8]: use cockroachdb/errors

File: pkg/server/environment_create.go [hunk 3]
```diff
@@ -27,32 +24,31 @@ import (
...
-		return NewError(http.StatusInternalServerError, err, "pglabel.set")
-	}
-	_, err = s.Queries.CreateImageInfo(context.Background(),
-		query.CreateImageInfoParams{OwnerToken: it,
-			Name: meta.Name, Digest: meta.Digest, Created: meta.Created, Size: meta.Size, Labels: pglabel})
-	if err != nil {
-		return NewError(http.StatusInternalServerError, err, "db.create-image")
+	if meta == nil {
+		return NewError(http.StatusInternalServerError,
+			errors.New("meta is nil"), "image-service.create-image")
```
Comment (kemingy) [inline pkg/server/environment_create.go:43]: This error message is not straightforward for end users.

Comment (gaocegege) [reply pkg/server/environment_create.go:43]: Do you mean meta is nil?

Comment (kemingy) [reply pkg/server/environment_create.go:43]: Yes

**Comment by gaocegege** [comment]
Comments are addressed, please take a look.
Thanks for your time :smile:

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**